mssql/.env.example

```diff
@@ -0,0 +1,2 @@
+SPICE_MSSQL_CONNECTION_STRING=
```

Comment:
This can just be `.env` because people can override with `.env.local` (and you can tell them to do that).

Reply:
Done. Also fixed this in the Github quickstart